src/styles/blocks/about.scss

```diff
@@ -0,0 +1,28 @@
+.about {
+  &__title {
+    margin: 0 0 32px;
+    font-weight: 500;
```

Comment:
u can add variable here, because u use it more than one time

Reply:
i agree with you